src/dosbox.cpp

```diff
@@ -584,6 +583,13 @@ void DOSBOX_Init(void) {
 	                  "scan2x, scan3x, tv2x, tv3x, sharp (default).");
 #endif
 
+	pstring = secprop->Add_path("maxresolution", always, "auto");
+	pstring->Set_help(
+	        "Optionally restricts the viewport resolution within the window/screen:\n"
+	        "  auto:      The viewport fills the window/screen (default).\n"
+	        "  <custom>:  Set max viewport resolution in W,H format.\n"
+	        "             For example: 960,720");
```

Comment:
the W,H format wasn't accepted:

![2021-09-22_06-46](https://user-images.githubusercontent.com/1557255/134369235-c7994d89-7755-455a-ba69-2ce811932f59.png)

(Probably a copy and paste issue, as the actual code uses WxH format like the other resolution, which is what we want).

Reply:
My bad, fixed.